File: ejercicios-js/ejercicio5.js
const num = Math.floor(Math.random() * 100) + 1;

function esPrimo(n) {
  if (n <= 1) return false;
  for (let i = 2; i <= Math.sqrt(n); i++) {
    if (n % i === 0) return false;
  }
  return true;
}

console.log(`Número generado: ${num}`);
console.log(esPrimo(num) ? `El número ${num} es primo` : `El número ${num} no es primo`);

File: ejercicios-py/ejercicio5.py
import random

numero = random.randint(1, 100)
es_primo = numero > 1

for i in range(2, numero):
    if numero % i == 0:
        es_primo = False
        break

print(f"El número {numero} {'es primo' if es_primo else 'no es primo'}")